.github/workflows/docker.yaml

```diff
@@ -43,4 +44,4 @@ jobs:
           context: "${{ matrix.app.dir }}/${{ matrix.app.name }}"
           push: true
           platforms: linux/amd64,linux/arm64
-          tags: ghcr.io/kong/kongair-${{ matrix.app.name }}:latest
+          tags: ghcr.io/battlebyte/kongair-${{ matrix.app.name }}:latest
```

Comment:
@battlebyte intentional change?

Reply:
Yes, once you fork the repo you have to change the URL to point to your ghcr. This should be changed back to "kong" now.

Reply:
Maybe there is a solution with variables that can be set per repo?  I'm not aware of what's possible for these

Reply:
I guess ${{ github.actor }} could be used, but not completely sure.

Reply:
@battlebyte can you revert this change or see if there is a solution with variables?  I don't think we should merge this in under your username.

Reply:
${{ github.actor }} should work here.